test/DomainOwnershipManager.test.js

```diff
@@ -0,0 +1,143 @@
+const {expect} = require("chai");
+const {ethers} = require("hardhat");
+const {loadFixture} = require("@nomicfoundation/hardhat-network-helpers");
+describe("DomainOwnershipManager", function () {
+    async function deployContract() {
+        const [owner, nonOwner] = await ethers.getSigners();
+        const DomainOwnershipManager = await ethers.getContractFactory("DomainOwnershipManager");
+        const domainOwnershipManager = await DomainOwnershipManager.deploy();
+        return {domainOwnershipManager, owner, nonOwner};
+    }
+
+    describe("Register Domain", function () {
+        it("Should register a new domain", async function () {
+            const {domainOwnershipManager, owner} = await loadFixture(deployContract);
```

Comment:
line 14 
as good practice, to load it once and use it in all tests
// - 1 point